test/account_objects-test.js

```diff
@@ -0,0 +1,275 @@
+  var objects_stepped = [];
+
+  test('stepped 1 at a time using marker/limit', function(done) {
+    var steps = [
+      function first_ripple_state(next) {
+        request_account_objects('bob', {limit: 1}, function(e, m){
+          var objects = m.account_objects;
+          var ripple_state = m.account_objects[0];
+
+          assert.equal(m.limit, 1);
+          assert.equal(objects.length, 1);
+          assert.equal(ripple_state.LedgerEntryType, 'RippleState');
+          assert.equal(ripple_state.HighLimit.issuer, bob);
+          assert.equal(ripple_state.LowLimit.issuer, G1);
+
+          objects_stepped.push(ripple_state);
+          next(null, m.marker);
+        });
+      },
+      function second_ripple_state(resume_marker, next) {
+        request_account_objects('bob', {limit: 1, marker: resume_marker},
+                                                          function(e, m){
+          var objects = m.account_objects;
+          var ripple_state = m.account_objects[0];
+
+
+          assert.equal(m.limit, 1);
+          assert.equal(objects.length, 1);
+          assert.equal(ripple_state.LedgerEntryType, 'RippleState');
+          assert.equal(ripple_state.HighLimit.issuer, bob);
+          assert.equal(ripple_state.LowLimit.issuer, G2);
+
+          objects_stepped.push(ripple_state);
+          next(null, m.marker);
+        });
+      },
+      function first_offer(resume_marker, next) {
+        request_account_objects('bob', {limit: 1, marker: resume_marker},
+                                                          function(e, m){
+          var objects = m.account_objects;
+          var offer = m.account_objects[0];
+
+          assert.equal(m.limit, 1);
+          assert.equal(objects.length, 1);
+
+          assert.equal(offer.LedgerEntryType, 'Offer');
+          assert.equal(offer.TakerGets.issuer, bob);
+          assert.equal(offer.Account, bob);
+
+          objects_stepped.push(offer);
+          next(null, m.marker);
+        });
+      },
+      function second_offer(resume_marker, next) {
+        request_account_objects('bob', {limit: 1, marker: resume_marker},
+                                                          function(e, m){
+          var objects = m.account_objects;
+          var offer = m.account_objects[0];
+
+          // assert.equal(m.limit, 1);
+          assert.equal(objects.length, 1);
+
+          assert.equal(offer.Account, bob);
+          assert.equal(offer.LedgerEntryType, 'Offer');
+          assert.equal(offer.TakerGets.issuer, G1);
+
+          assert.equal(m.marker, undefined);
+          objects_stepped.push(offer);
+          next();
+        });
+      },
+    ];
+
+    async.waterfall(steps, function (err, result) {
+      assert.ifError(err);
+      done();
+    });
+  });
+
+  test('unstepped == stepped', function(done) {
+    if (objects_stepped.length === 0)  {
+      return done(new Error("stepped failed"));
```

Comment:
This is an edge case and pretty nickpicky, but if I'm debugging, and want to run only this test (eg. `test.only`), that's not really possible, because it depends on the other test finishing first. (It also depends on the tests running in order, but I think that's guaranteed by mocha.)  Is it worth calling the one-at-a-time test here before flat out failing?